docs/rules/no-empty-alternative.md

```diff
@@ -40,7 +42,7 @@ Nothing.
 
 ## :heart: Compatibility
 
-This rule was taken from [eslint-plugin-clean-regex].  
+This rule was taken from [eslint-plugin-clean-regex].
```

Comment:
This is a bit of a problem. Ending a line with 2 spaces indicates a line break in markdown, so we can't always trim trailing spaces.

Does eslint-doc-generator do this automatically? If so, this needs to be fixed. I think trimming trailing spaces is good, but it needs to be careful about line breaks.

Reply:
Interesting. Using trailing spaces for linebreaks seems very fragile as many people including myself have their editors set to automatically trim trailing spaces. So this change came from my editor, not eslint-doc-generator. I can revert this change, but I would suggest using a trailing backslash `\` as a less-fragile and more obvious way to force a linebreak.

Reply:
Uhhh, I really like the trailing `\` solution, never knew this existed. I actually also have me editor setup to trim trailing spaces and needed to be careful not to commit those changes every time.

Then let's change the `<space><space>` to `<space><backslash>`.

Reply:
I don't think there's any need for a space before the backslash. Here's my separate PR to fix this among other things: https://github.com/ota-meshi/eslint-plugin-regexp/pull/494

Reply:
I just wanted a space because I think it would look nicer. But not having it is okay too.